generators/app/templates/common/README.md

```diff
@@ -87,6 +95,18 @@ Set up your AWS account locally to be able to access the different resources:
   - *(Optional)* In your IDE, modify the default terminal env variables of your project to add `AWS_PROFILE=<%= projectSlug %>`.
   It allows you to use the right AWS profile when calling Python files.
 
+<% } -%>
+<% if (includeApi && apiInfrastructure === "gcp") { -%>
+### Setup GCP for your project
+Set up your GCP account locally to be able to access the different resources:
+- Run `gcloud auth login` and follow the instructions to log in to your GCP account.
+- Set the GCP project ID:
+  ```bash
+  gcloud config set project <%= gcpProjectId %>
```

Comment:
typo: <%= gcpProjectId %**=**>
```suggestion
  gcloud config set project <%= gcpProjectId %=>
```

Reply:
No, it's correct ;)
<img width="469" alt="image" src="https://github.com/sicara/sicarator/assets/10354968/e7133115-5516-4cb0-ab58-7a5e99f67eb3">